.github/workflows/elixir.yml

```diff
@@ -25,8 +25,8 @@ jobs:
       - name: Setup Elixir
         uses: erlef/setup-beam@v1
         with:
-          elixir-version: '1.10.4'
-          otp-version: '23.0.4'
+          elixir-version: "1.10.4"
```

Comment:
Out of the scope of this PR: the CI should run in all the (major) supported Elixir versions.